.github/workflows/ios_deploy_staging_to_firebase.yml

```diff
@@ -9,9 +9,6 @@ jobs:
   build_and_upload_staging_app_to_firebase:
     name: Build And Upload iOS Staging Application To Firebase
     runs-on: macOS-latest
-    defaults:
-      run:
-        working-directory: template
```

Comment:
I think you're missing this request https://github.com/nimblehq/flutter-templates/pull/153#issuecomment-1517235024 🙏